rudder-agent/SOURCES/debug-script/tests/rudder_cf_serverd_logs

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh 
+#
+# cf-serverd logs 
+#
+# This test allows to display the logs related to cf-served
+#
+
+# Declare your tests here (KEEP "${TESTS}")
+TESTS="${TESTS} cf_serverd logs"
+
+# Descrive your tests here
+cfserverd_logs_description="display cf-serverd related to cf-serverd file"
+
+cf_serverd() {
+
+  # BY default, this test is successful
+  # CF_SERVERD_LOGS='ok'
+
+  RUDDER_LOG_PATH="/var/log/syslog"
```

Comment:
Did you check that this file also contains cf-serverd log on redhat ? i think it's a diferent one